docs/pages/reference/backends.mdx

```diff
@@ -182,7 +187,7 @@ teleport:
   in Teleport Architecture documentation.
 </Admonition>
 
-S3 buckets can only be used as storage for the recorded sessions. S3 cannot
+S3 and S3 compatible buckets can only be used as storage for the recorded sessions. S3 cannot
 store the audit log or the cluster state.
```

Comment:
Note, this is technically no longer true, as we now have an S3 based audit log too.